### PR TITLE
Adds sample code for RaisedButton widget

### PR DESCRIPTION
**Pull request by laertispappas**
Addresses https://github.com/flutter/flutter/issues/12382 and https://github.com/flutter/flutter/issues/21136

### CODE REVIEW

**Comment by laertispappas** [comment]
I signed it!

File: packages/flutter/lib/src/material/raised_button.dart [hunk 1]
```diff
@@ -32,6 +32,81 @@ import 'theme_data.dart';
...
+///                 ),
+///               ],
+///             ),
+///           ],
+///         ),
+///       ),
+///     );
+///   }
+/// }
+
```
Comment (Hixie) [inline packages/flutter/lib/src/material/raised_button.dart]: nit: extraneous blank line

File: packages/flutter/lib/src/material/raised_button.dart [hunk 2]
```diff
@@ -32,6 +32,81 @@ import 'theme_data.dart';
...
+///
```
Comment (Hixie) [inline packages/flutter/lib/src/material/raised_button.dart]: nit: also here

**Comment by Hixie** [comment]
This looks great, just some minor nits!

When you update this you'll probably find it complains about MyHomePage being already used. You'll need to change the class name to be different than in any other example, e.g. CounterPage or something.

**Comment by laertispappas** [comment]
@Hixie  thanks for the feedback. I will follow up.

**Comment by laertispappas** [comment]
@Hixie  I have updated the code. Wanna have alook? Thank you in advance.

**Comment by HansMuller** [comment]
Please accept my apologies for not getting your contribution landed. I'm afraid it got lost in the final push for Flutter 1.0. That's no excuse really, sorry about dropping it.

Since then another developer has contributed a RaisedButton sample, https://github.com/flutter/flutter/pull/27169/, which will serve as well. 

I'm going to close this PR. I hope this will not discourage you from future contributions!

**Comment by laertispappas** [comment]
@HansMuller thanks for the update and your message. It's okay I understand. I hope I get more involved in the near future.

**Pull request closed without merging**